.woodpecker.yml

```diff
@@ -239,7 +237,9 @@ steps:
     settings:
       repo: dessalines/lemmy
       dockerfile: docker/Dockerfile
-      platforms: linux/amd64,linux/arm64
+      # TODO fix arm build: see: https://woodpecker.join-lemmy.org/repos/129/pipeline/2888/20
```

Comment:
See this run for why arm builds are broken with this change: ring isn't compiling on arm. I tried with a few fixes linked, but none of them worked.